frontend/src/app/components/wp-table/timeline/cells/timeline-cell-renderer.ts

```diff
@@ -396,7 +396,8 @@ export class TimelineCellRenderer {
     let selectionMode = renderInfo.viewParams.activeSelectionMode;
 
     // Cannot edit the work package if it has children
-    if (!wp.isLeaf && !selectionMode) {
+    // and it is not on 'Manual scheduling' mode
+    if (!wp.isLeaf && !selectionMode && !wp.scheduleManually) {
```

Comment:
Might be just my preference but having to negate all the individual elements of an "and" joined condition typically indicates that you can improve readability by turning it into an "or" joined condition and either negate the whole condition or even better, switch the cases: 
```
if (wp.isLeaf || selectionMode || wp.scheduleManually) {
  bar.classList.remove('-readonly');
} else {
  bar.classList.add('-readonly');   
}
```

This could then also be reflected in the comment which would no longer have to rely on a duplicate negation ("Cannot ... is not on")